docs/connected_orgs.rst

```diff
@@ -1,14 +1,76 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
 
-Connect a production org
-------------------------
 
-Connect a sandbox org
----------------------
 
-Global orgs
------------
+The ``org connect`` Command
+---------------------------
+To connect to a persistent org use::
 
-Use a custom connected app
+    $ cci org connect <org_name>
+
+This will automatically open a browser window pointed to a Salesforce login page.
+The provided ``<org_name>`` will be the name that CumulusCI associates with org you log into.
+
+If you're connecting to a sandbox use the ``--sandbox`` flag.
+
+By default, ``cci org connect`` stores the OAuth credentials for connected orgs in a *project-specific* keychain.
+This means that an org connected while working in Project A's repository, will not be available while working in Project B's repository.
+If you would like to make the org available to *all* CumulusCI projects use the ``--global`` flag.
+
+.. note::
+    Connecting an org via ``cci org connect`` does *not* expose that org to ``SFDX``.
+
+
+Verify Your Connected Orgs
 --------------------------
+You can use ``cci org list`` to look for your org listed under the "Connected Org" table.
+
+The following shows a single persistent org connected to CumulusCI with the name "devhub".
+
+.. image:: images/connected_org.png
```

Comment:
I'd suggest showing the output in a code-block unless there's a compelling reason to use an image. Easier to maintain.